tests/tt_metal/tt_metal/perf_microbenchmark/dispatch/test_prefetcher.cpp

```diff
@@ -880,18 +880,19 @@ void gen_packed_read_test(Device *device,
                           vector<uint32_t>& prefetch_cmds,
                           vector<uint32_t>& cmd_sizes,
                           DeviceData& device_data) {
-
+    static constexpr uint32_t min_read_size = 128;
```

Comment:
where does 128 come from?

Reply:
128 was from the original code. If the length was less than 128 then it set the length to 128. I extracted it out to increase readability.